app/src/services/OrderBookService.js

```diff
@@ -1,8 +1,12 @@
-import {httpClient} from './HttpClient';
-
-export default class OrderBookService {
-
-    static getOrderBookPrices = (instrument, timestamp) => {
-        return httpClient.get(`/orderbook/${instrument}/${timestamp}`);
-    };
-};
\ No newline at end of file
+import {httpClient} from './HttpClient';
+
+export default class OrderBookService {
+
+    static getOrderBookPrices = (instrument, timestamp) => {
+        return httpClient.get(`/orderbook/${instrument}/${timestamp}`);
+    };
+
+    static getOrderBookPricesByMessageOffset = (instrument, timestamp, offset) => {
```

Comment:
Maybe we should use price _levels_ here? Something like `getPriceLevelsByMessageOffset`